flash/attrs.go

```diff
@@ -0,0 +1,53 @@
+package flash
+
+import "fmt"
+
+type JedecID struct {
+	ManufID  uint8
+	MemType  uint8
+	Capacity uint8
+}
+
+func (id *JedecID) AsUint32() uint32 {
```

Comment:
I think just calling this `Uint32` is more Go style. See for example [reflect](https://godoc.org/reflect#Value.Uint).

Reply:
Done